app/models/badges/insider_badge.rb

```diff
@@ -0,0 +1,14 @@
+module Badges
+  class InsiderBadge < Badge
+    seed "Insider",
+      :ultimate,
+      'insider',
+      'Member of Exercism Insiders'
+
+    def award_to?(user) = VALID_INSIDERS_STATUSES.include?(user.insiders_status)
+    def send_email_on_acquisition? = true
+
+    VALID_INSIDERS_STATUSES = %i[active active_lifetime].freeze
```

Comment:
The majority of people will get this via the `active` status, but I added `active_lifetime` for those initial users that immediately become lifetime insiders.